src/FilesAndStreams/Constants/Messages.cs

```diff
@@ -0,0 +1,202 @@
+﻿namespace FilesAndStreams.Constants;
+
+/// <summary>
+/// Constants used as messages.
+/// </summary>
+public static class Messages
+{
+    /// <summary>
+    /// To display title of app.
+    /// </summary>
+    public const string FileAndStreams = "Files and streams";
+
+    /// <summary>
+    /// To display available tasks.
+    /// </summary>
+    public const string TaskOptions = "1. Task 1\n2. Task 2\n3. Task 3\n4. Task 4\n5. Exit";
+
+    /// <summary>
+    /// To display title of task
+    /// </summary>
+    public const string TaskTitle = "Task {0}";
+
+    /// <summary>
+    /// To display available options.
+    /// </summary>
+    public const string Task1MenuPrompt = "1. Create sample files\n2. Read sample files\n3. Process sample files\n4. Exit";
+
+    /// <summary>
+    /// To display available options.
+    /// </summary>
+    public const string Task2MenuPrompt = "1. Create sample files\n2. Read sample files\n3. Process sample files\n4. Exit";
+
+    /// <summary>
+    /// To display prompt to the user.
```

Comment:
```suggestion
    /// Prompt message displayed to the user to enter their desired action.
```
Update in all the other constants. Don't have generic documents for the constantans. Be precise